cdap-data-fabric/src/main/java/co/cask/cdap/data2/util/hbase/HTableNameConverter.java

```diff
@@ -86,6 +86,14 @@ protected static String getHBaseTableName(CConfiguration cConf, TableId tableId)
    */
   public abstract String getSysConfigTablePrefix(String hTableName);
 
+  /**
+   * Returns the {@link TableId} for the given hbase tablename
+   * Note: Sub-classes should override this method for their version specific implementation.
```

Comment:
I simply don't think its needed because it should be obvious - the method is marked `abstract`.